pkg/assets/assets.go

```diff
@@ -0,0 +1,67 @@
+package assets
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/coreos/bootkube/pkg/assets/internal"
+)
+
+const (
+	AssetPathCAKey           = "ssl/ca.key"
```

Comment:
Yeah, I'm on board for changing across projects (just hadn't gotten around to it elsewhere -- somewhat low priority). I'm indifferent towards tls/ssl fine to change it (I refer to it as tls everywhere else in code).